### PR TITLE
[no ticket][risk=no] Puppeteer fix nightly tests

### PR DESCRIPTION
**Pull request by aweng98**
CircleCi [failure](https://app.circleci.com/pipelines/github/all-of-us/workbench/8878/workflows/8007c2e1-e329-40c1-a8e5-1e467862f23c/jobs/129727).
Failure were caused by waiting for the runtime status spinner to stop in this [call](https://github.com/all-of-us/workbench/blob/283b78a4e82e3e12fa558a95be4eabcad4c722ea/e2e/app/page/notebook-page.ts#L255). The fix is to exclude runtime status spinner while for page to load.

ran on localhost:

Screen Shot 2021-04-02 at 11 44 06 AM

### CODE REVIEW

File: e2e/package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "test-staging:debug": "cross-env WORKBENCH_ENV=staging PUPPETEER_HEADLESS=false DEBUG=true yarn _test --maxWorkers=1",
     "test:ci": "yarn _test --ci --maxWorkers=2 --forceExit",
     "test:ci:debug": "yarn _test --ci --debug --maxWorkers=1 --bail 1",
-    "test-nightly": "cross-env TEST_MODE=nightly-integration yarn _test --maxWorkers=1",
+    "test-nightly": "cross-env WORKBENCH_ENV=test TEST_MODE=nightly-integration yarn _test --maxWorkers=2",
```
Comment (aweng98) [inline e2e/package.json:23]: unrelated to fix. Making tests run faster.

File: e2e/tests/nightly/dataproc-to-gce.spec.ts [hunk 2]
```diff
@@ -87,5 +87,6 @@ describe('Updating runtime compute type', () => {
     // Delete notebook
     const workspaceAnalysisPage = await notebookPreviewPage.goAnalysisPage();
     await workspaceAnalysisPage.deleteResource(notebookName, ResourceCard.Notebook);
+    await workspaceAnalysisPage.deleteWorkspace();
```
Comment (aweng98) [inline e2e/tests/nightly/dataproc-to-gce.spec.ts:90]: test should have deleted workspace too.

**Comment by aweng98** [comment]
@petesantos this test failure could happen again in nightly-run if fix is not merged. Can you review today? thank you.

**Review by petesantos — APPROVED** [review]
